dashboard/app/models/stage.rb

```diff
@@ -14,6 +14,7 @@
 # (Intended to replace most of the functionality in Game, due to the need for multiple app types within a single Game/Stage)
 class Stage < ActiveRecord::Base
   has_many :script_levels, -> { order('position ASC') }, inverse_of: :stage
+  has_one :plc_learning_module, class_name: 'Plc::LearningModule', dependent: :destroy
```

Comment:
Let's leave off the dependent: destroys for now. If its easy to delete a script/stage, or if there's a bug in this, we risk losing a whole lot of content and student progress.